src/libs/API/parameters/CreateWorkspaceTagsParams.ts

```diff
@@ -0,0 +1,10 @@
+type CreateWorkspaceTagsParams = {
```

Comment:
Did we agree we will keep calling this Workspace or should we also update this to Policy (also for categories) @luacmartins

Reply:
Updated to policy